course-v2/layouts/partials/course_info.html

```diff
@@ -4,9 +4,9 @@
 <div class="course-info">
   <div class="course-detail-title {{ if $inPanel }}bg-light-gray{{ end }}">
     <div class="d-flex">
-      <div class="font-black {{ if $inPanel }}py-1 pl-3 mt-2 m-0{{ end }}">
+      <h4 class="font-black {{ if $inPanel }}py-1 pl-3 mt-2 m-0{{ end }}">
```

Comment:
It would probably be better to use an h2 or h3 for this and style it as a smaller title if that's what we want.

Headings are used by some viewers to generate outlines, and currently this h4 will be grouped under the h3's in the content:

<img width="267" alt="Screenshot 2023-05-16 at 11 56 11 AM" src="https://github.com/mitodl/ocw-hugo-themes/assets/9010790/b230d036-fd56-48f7-8e84-42e3a014036d">

Reply:
That screenshot is from https://chrome.google.com/webstore/detail/headingsmap/flbjommegcjonpdmenkdiocclhjacmbi/related, btw. I thought I'd previously used VoiceOver on macs to generate ToCs like this, but I can't seem to figure out how to do it right now.